src/utils/date.ts

```diff
@@ -16,14 +16,14 @@ export const convertUtcTimestampToLocal = (utcTimestamp: number) => {
   return dateUTC.unix()
 }
 
-export const convertTimestampWithMoment = (utcTimestamp: number) => {
+export const faketimer = (utcTimestamp: number) => {
   return dayjs
     .utc(utcTimestamp * 1000)
     .tz(dayjs.tz.guess())
     .format('MMM D, hh:mm z')
 }
 
-export const calculateTimeDifference = (utcTimestamp: number) => {
+export const calculatefaketimer = (utcTimestamp: number) => {
```

Comment:
`camelCase`

Reply:
These functions are not used any. I will remove them